README.md

```diff
@@ -297,12 +297,18 @@ This sends all comment notifications to a Mattermost channel, including public a
 
 **Note**: If you experience problems with Jira-related user interactions in Mattermost such as creating issues, disable these features by setting **Allow users to connect their Mattermost accounts to Jira** to false in **System Console > Plugins > Jira**. This setting does not affect Jira webhook notifications. After changing this setting to false, disable, then re-enable this plugin in **System Console > Plugins > Plugin Management** to reset the plugin state for all users.
 
+Sometimes the plugin may crash unexpectedly, you may notice:  
+
+- you will see a response in red text below the chat window displaying ` slash command with trigger of  '/(name)' not found,`  - If you check your log file, look for messages that refer to "plugins" and "health check fail", "ExecuteCommand" etc. 
+
+If you encopunter these types of issues - you can set `LogSettings.FileLevel` to `DEBUG` in your config.json settings.   This will enable debug logging and give more verbose error events in the system log. Then try re-enabling the plugin in the system-console. These log results may be requested by others in the forum or by our support team.
```

Comment:
Remove section about log level (discussed offline)

Reply:
@levb See the above suggestion. This came from this discussion: https://community-release.mattermost.com/core/pl/chu9scfttbbaxbgohx5yyfyk1y

It was said that we may not want to advise customers to use the DEBUG log level, but we've included it in our docs here.